lib/art_vandelay.rb

```diff
@@ -177,8 +178,18 @@ def build_csv(csv_string, headers)
     def build_params(row, attributes)
       attributes = attributes.stringify_keys
 
-      row.to_h.stringify_keys.transform_keys do |key|
-        attributes[key] || key
+      if strip
+        params = row.to_h.stringify_keys.transform_keys do |key|
+          attributes[key.strip] || key.strip
+        end
+
+        params.transform_values do |value|
+          value.strip
+        end
```

Comment:
How would you feel about using [tap](https://rubyapi.org/3.2/o/s?q=tap) to consolidate these two mappings? I don't feel too strongly about it, but find it reads a little better. What do you think?

```suggestion
        row.to_h.stringify_keys.transform_keys do |key|
          attributes[key.strip] || key.strip
        end.tap do |new_params|
          new_params.transform_values!(&:strip)
        end
```

Reply:
I like it. I don't reach for `tap` enough.